### ISSUE TITLE
Use `caption` instead of `shorttext`

### ISSUE THREAD

**Issue by iilyak** [reidgould/tiddlywiki-dropboard#7]
Awesome plugin. Are there any downsides to replace use of a `shorttext` field with `caption`. Tiddlywiki have special logic to handle `caption` field. In particular it is displayed correctly in TOCs and in results produced by `list` filters. So I ended up providing the same text in two fields `shorttext` to make DropBoard happy and `caption` field to make rest of Tiddlywiki happy. However in order to do it there are way too many clicks required:
- drag "New Tiddler" into column
- type shorttext and text
- drag created card onto "open in main story"
- scroll up where the tiddler is opened
- click "edit the tiddler"
- type `caption` in `Add new field`
- type the same text as in `shorttext` in the `value` of the field
- click `add`
- save tiddler
 
Which is very inconvenient. I would prefer to enter `caption` when I drug "New Tiddler" draggable into column. I am afraid that changing the https://github.com/reidgould/tiddlywiki-dropboard/blob/gh-default/Source/editions/dropboard-plugin/plugins/dropboard/tiddlers/templates/edit/Card.tid#L49 line wouldn't be enough and it might be more complex change.

**Comment by linonetwo** [comment]
And that `shorttext` field can't be searched currently, make it quite dangerous to use.

**Comment by gMakaan** [comment]
Yes, there are issues, but when using `caption` instead - you lose full featured text formatting on this "short text".
